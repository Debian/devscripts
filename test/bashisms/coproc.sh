#!/bin/sh

coproc true # BASHISM
