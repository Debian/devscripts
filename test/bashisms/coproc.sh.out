possible bashism in bashisms/coproc.sh line 3 (coproc):
coproc true # BASHISM
